samples/sensor/hx711/prj.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_SENSOR=y
+CONFIG_HX711=y
+CONFIG_GPIO=y
+CONFIG_HX711_CALIBRATION_WEIGHT_INTEGER=500
+CONFIG_HX711_CALIBRATION_WEIGHT_DECIMAL=500000
+CONFIG_LOG=y
+CONFIG_REBOOT=y
```

Comment:
if you need PM, you'll need to turn on `PM_DEVICE` and `PM_DEVICE_RUNTIME` in this case.

Reply:
Done.